modules/sdk-coin-apt/src/lib/utils.ts

```diff
@@ -90,12 +97,61 @@ export class Utils implements BaseUtils {
     }
   }
 
+  validateRecipients(addressArg: EntryFunctionArgument, amountArg: EntryFunctionArgument): boolean {
```

Comment:
combine `validateRecipients` and `fetchAndValidateRecipients` methods into `validateRecipients()` with return type `{ recipients: { deserializedAddresses: string[]; deserializedAmounts: Uint8Array[] }; isValid: boolean }`